packages/client/modules/demo/components/CreateAccountSection.tsx

```diff
@@ -124,6 +124,7 @@ const CreateAccountSection = (props) => {
         <td style={buttonCellStyle}>
           <a
             data-cy='create-account'
+            role='button'
```

Comment:
Since this anchor has button styling, it's nice to provide a `role` for screen readers. It also made it easier to identify in test.